### PR TITLE
no need to mention ownCloud explicitely in the versions and trash bin app

### PR DESCRIPTION
**Pull request by schiessle**
please review @blizzz @MorrisJobke @LukasReschke

### CODE REVIEW

**Comment by MorrisJobke** [comment]
Makes sense 👍

File: apps/files_versions/appinfo/info.xml [hunk 1]
```diff
@@ -5,8 +5,8 @@
...
-In addition to the expiry of versions, ownCloud’s versions app makes certain never to use more than 50% of the user’s currently available free space. If stored versions exceed this limit, ownCloud will delete the oldest versions first until it meets this limit. More information is available in the Versions documentation. 
+	This application automatically maintain older versions of files that are changed. When enabled, a hidden versions folder is provisioned in every user’s directory and is used to store old file versions. A user can revert to an older version through the web interface at any time, with the replaced file becoming a version. The app automatically manages the versions folder to ensure the user doesn’t run out of Quota because of versions.
```
Comment (vincchan) [inline apps/files_versions/appinfo/info.xml]: @schiessle either "maintains" or "This application enables the app to" :)

Comment (schiessle) [inline apps/files_versions/appinfo/info.xml]: fixed :smile:

**Comment by LukasReschke** [comment]
LGTM

**Comment by schiessle** [comment]
@LukasReschke you LGTM bot doesn't like this PR... Maybe he accept my :+1: :wink:

**Comment by schiessle** [comment]
Worked :grinning:

**Pull request merged**